src/Storages/IStorage.h

```diff
@@ -585,7 +585,7 @@ class IStorage : public std::enable_shared_from_this<IStorage>, public TypePromo
     /// Returns true if all disks of storage are read-only.
     virtual bool isStaticStorage() const;
 
-    virtual bool isColumnOriented() const { return false; }
+    virtual bool supportsSamplingColumns() const { return false; }
```

Comment:
Maybe it is better to name `supportsSubsetOfColumns` or similar? Since it is easy to mix sampling with `SAMPLE BY` (especially since there is no comment in the interface).

Reply:
Yep, it will be much better, thanks